topics/chrome/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: google-chrome
+related: firefox,safari,edge,opera,chromium,browser,chrome-extension
+created_by: Google
+display_name: Chrome
+github_url: https://github.com/googlechrome
+logo: chrome.png
+released: September 2, 2008
+short_description: Chrome is a web browser from the tech company Google.
+topic: chrome
+url: https://www.google.com/chrome/
+---
+
+Chrome is the most popular web browser worldwide as of mid-2017, made by the tech company Google. It's available for most operating systems including Windows, macOS and Linux and on multiple platforms such as the desktop, phones and tablets.
```

Comment:
Can you add commas before `and Linux` and `and tablets` to suit the style guide? See https://github.com/github/explore/blob/master/docs/styleguide.md#commas.